test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -167,6 +167,30 @@
                 (is (= 403 (:status response))
                     (ks/pprint-to-string response))))))))))
 
+(deftest ^:integration certificate-with-ca-true-extension-refused
+  (testing (str "Validates that the server rejects a csr for signing"
+                " that has the v3 CA:TRUE extension")
+    (bootstrap/with-puppetserver-running
+      app
+      {:jruby-puppet
+       {:master-conf-dir (str "./dev-resources/puppetlabs/ca_true_test/master/"
+                              "conf")}}
+       (let
+         [master-conf-dir (str "./dev-resources/puppetlabs/ca_true_test/"
```

Comment:
You could wrap the outer `bootstrap/with-puppetserver-running` with a let block:

```clojure
(let [master-conf-dir "./dev-resources/..."]
  (bootstrap/with-puppetserver-running
    app
    {:jruby-puppet {:master-conf-dir master-conf-dir}}
...
```
that's probably the easiest way.